src/app/admin/login/page.js

```diff
@@ -0,0 +1,63 @@
+"use client";
+import React, { useState } from "react";
+import { Form, Button, Alert, Container } from "react-bootstrap";
+import { signInWithGoogle } from "../../../../utils/supabase/actions";
+import { createClient } from "../../../../utils/supabase/client";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setLoading(true);
+
+    const supabase = await createClient();
+    const response = await supabase.from('approved_admin_emails').select('*').eq('email', email).single(); // checks if the email from the form is a approved admin
+
+    if ( response.error ) {
+      setError(true);
+      setErrorMessage("You are not a registered admin")
+    }
+
+    else {
+      await signInWithGoogle();
+    }
+
+    setLoading(false);
+  };
+
+  return (
+      <Container>

```

Comment:
PRAISE: Nice job wrapping components (even a Form) in a Bootstrap Container!

I was just reading the other day that Bootstrap Containers can help with the stickiness of navbars and headers, and have other properties!